docs/API-Reference/bpmn-js/importXML.md

```diff
@@ -10,3 +10,62 @@ grand_parent: "API Reference"
 ---
 
 # importXML
+
+`importXML` API parses and renders a BPMN 2.0 diagram.
+
+## Usage
+
+importXML(xml: _[String](#)_, bpmnDiagram: _[ModdleElement](#)_ or _[String](#)_): Promise<_[ImportXMLResult](#importxmlresult)_, _[ImportXMLError](#importxmlerror)_>
+
+* `xml`: The BPMN 2.0 XML.
+* `bpmnDiagram`: The BPMN diagram or the id of the diagram to render _(if not provided, the first one will be rendered)_.
```

Comment:
See above.

BPMN diagram moddle element to render **or** id string of the diagram to render.

(I am not sure I understand this second parameter)

Reply:
Updated the explanation. It seems like some BPMN XML could contain multuple diagrams, so in this case this 2nd parameter helps us to specify which diagram to import.